acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/LoggingTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright contributors to Hyperledger Besu
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.tests.acceptance;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.hyperledger.besu.tests.acceptance.dsl.AcceptanceTestBase;
+import org.hyperledger.besu.tests.acceptance.dsl.WaitUtils;
+import org.hyperledger.besu.tests.acceptance.dsl.node.BesuNode;
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+public class LoggingTest extends AcceptanceTestBase {
+
+  @Test
+  public void testDefaultLoggingIsAtLeastInfo() throws IOException {
+    final BesuNode node = besu.runCommand("");
+    cluster.startConsoleCapture();
+    cluster.runNodeStart(node);
+
+    // Wait for log to contain an info level
+    WaitUtils.waitFor(
+        10, () -> assertThat(cluster.getConsoleContents()).doesNotContain("| INFO |"));
```

Comment:
Although the fix works, this test still passes even with the broken code...The test class itself still prints INFO logs.